File: TutoringApp/Data/Models/TutoringSession.cs
using System;
using TutoringApp.Data.Models.Base;
using TutoringApp.Data.Models.Enums;

namespace TutoringApp.Data.Models
{
    public class TutoringSession : BaseEntity
    {
        public DateTimeOffset CreationDate { get; set; }
        public bool IsSubscribed { get; set; }
        public DateTimeOffset SessionDate { get; set; }
        public TutoringSessionStatusEnum Status { get; set; }

        public DateTimeOffset? StatusChangeDate { get; set; }
        public TutoringSessionEvaluationEnum? Evaluation { get; set; }
        public string EvaluationComment { get; set; }
        public bool IsReminderSent { get; set; }

        public AppUser Student { get; set; }
        public string StudentId { get; set; }

        public AppUser Tutor { get; set; }
        public string TutorId { get; set; }
    }
}

File: TutoringApp/ClientApp/src/app/models/tutoring/tutoring-sessions/tutoring-session.ts
import { TutoringSessionEvaluationEnum } from '../../enums/tutoring-session-evaluation-enum';
import { TutoringSessionStatusEnum } from '../../enums/tutoring-session-status-enum';

export interface TutoringSession {
  id: number;
  moduleName: string;
  participantName: string;
  creationDate: Date;
  isSubscribed: boolean;
  sessionDate: Date;
  status: TutoringSessionStatusEnum;
  statusChangeDate: Date;
  evaluation: TutoringSessionEvaluationEnum;

  statusDisplay: string;
  evaluationDisplay: string;
  isActive: boolean;
}
